integration/local_server_list_test.go

```diff
@@ -0,0 +1,65 @@
+//go:build integration
+// +build integration
+
+/*
+ * Copyright (c) 2024-present Fabien Potencier <fabien@symfony.com>
+ *
+ * This file is part of Symfony CLI project
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package integration
+
+import (
+	"os/exec"
+	"regexp"
+	"strings"
+	"testing"
+)
+
+func TestServerList(t *testing.T) {
+	startCommands := []string{
+		"../symfony-cli server:start -d --dir=phpinfo/",
+		"../symfony-cli server:start -d --dir=hello_world/",
+	}
+
+	for _, command := range startCommands {
+		cmd := exec.Command(strings.Split(command, " ")[0], strings.Split(command, " ")[1:]...)
```

Comment:
to get more control I would rather directly use the code here (starting the server in a goroutine and issuing http request, worst case: starting the application instead of directly the HTTP server).

Reply:
Would you do this in all functional tests or only this one?